tests/integration_tests/cpu/propagator/covariance_transport.cpp

```diff
@@ -190,7 +190,7 @@ class detray_propagation_HelixCovarianceTransportValidation
             helix_intersector<typename destination_mask_type::shape,
                               algebra_type>{}(hlx, surface_descriptor<>{},
                                               mask_1, trf_1,
-                                              this->mask_tolerance));
+                                              std::array<scalar_type, 2u>{this->mask_tolerance,this->mask_tolerance}));
```

Comment:
Actually, there is an interface that allows to pass only one value, however, in that operator the intersection implementation is wrongfully called with two different values. E.g. in https://github.com/acts-project/detray/blob/9e7a67db9880fe5538bb6991211919d4d1440aba/core/include/detray/navigation/intersection/helix_cylinder_intersector.hpp#L199 it should read:
```
        return this->operator()(h, sf_desc, mask, trf, {mask_tolerance, mask_tolerance},
                                0.f);
```
Similar for the other two helix intersectors. For the ray intersectors, the 0 is correct

Reply:
Fixed